public/app/plugins/datasource/jaeger/CheatSheet.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+export default function CheatSheet() {
+  return (
+    <>
+      <h2 id="jaeger-cheat-sheet">Jaeger Cheat Sheet</h2>
+      <p>
+        Documentation for the Jaeger data source can be found{' '}
+        <a
+          href="https://grafana.com/docs/grafana/latest/datasources/jaeger"
+          target="_blank"
+          rel="noreferrer"
+          style={{ color: '#1F62E0' }}
+        >
+          here,
+        </a>{' '}
+        but here is a quick overview of the query types:
+      </p>
+
+      <hr />
+      <ul style={{ listStyle: 'none' }}>
```

Comment:
CSS should be extracted into a getStyles function

Reply:
Done. https://github.com/grafana/grafana/pull/54521/commits/409227f895e5aed4823e7b492bd8e145efed421a